### PR TITLE
Verify custom domain using TXT record

### PR DESCRIPTION
**Pull request by cpareek**
Verification using CNAME isn't working on the new frondoors hence we need to verify it using the TXT records.

### CODE REVIEW

File: source/cloud-native-platform/path-to-live/public-dns.html.md.erb [hunk 1]
```diff
@@ -17,32 +17,24 @@ follow the [shuttering](shutter.html#pre-requisite) instructions for which recor
...
-- name: "afdverify.${name}"
-  ttl: 300
-  record: "afdverify.{copy endpoint hostname from frontdoor}"
-- name: "${name}"
-  ttl: 300
-  record: "{copy endpoint hostname from frontdoor}"
+- name: "_dnsauth.${name}"
+  ttl: 3600
+  record: 
+    - "{copy TXT record from frontdoor -> Domains -> and then click Pending in the valildation state column}"
```
Comment (timja) [inline source/cloud-native-platform/path-to-live/public-dns.html.md.erb:33]:
```suggestion
    - "{copy TXT record from frontdoor -> Domains -> and then click Pending in the validation state column}"
```

File: source/cloud-native-platform/path-to-live/public-dns.html.md.erb [hunk 2]
```diff
@@ -17,32 +17,24 @@ follow the [shuttering](shutter.html#pre-requisite) instructions for which recor
...
 
 
 ```
-- name: "afdverify.${name}"
-  ttl: 300
-  record: "afdverify.{copy endpoint hostname from frontdoor}"
-- name: "${name}"
-  ttl: 300
-  record: "{copy endpoint hostname from frontdoor}"
+- name: "_dnsauth.${name}"
```
Comment (timja) [inline source/cloud-native-platform/path-to-live/public-dns.html.md.erb:30]: needs ignoring in the spelling file

File: source/cloud-native-platform/path-to-live/public-dns.html.md.erb [hunk 3]
```diff
@@ -17,32 +17,24 @@ follow the [shuttering](shutter.html#pre-requisite) instructions for which recor
...
+Here is an example [Pull Request](https://github.com/hmcts/azure-public-dns/pull/1409/files),
 - Create a PR with records as per the below convention
 (make sure you replace the variables with the correct values).
 
 
 ```
-- name: "afdverify.${name}"
-  ttl: 300
-  record: "afdverify.{copy endpoint hostname from frontdoor}"
-- name: "${name}"
```
Comment (timja) [inline source/cloud-native-platform/path-to-live/public-dns.html.md.erb:37]: this is still needed? but you may want to split it into multiple steps?

File: source/cloud-native-platform/path-to-live/public-dns.html.md.erb (hunk 2 above)
Comment (cpareek) [reply source/cloud-native-platform/path-to-live/public-dns.html.md.erb:30]: yupp done

File: source/cloud-native-platform/path-to-live/public-dns.html.md.erb (hunk 3 above)
Comment (cpareek) [reply source/cloud-native-platform/path-to-live/public-dns.html.md.erb]: added

File: .github/actions/spelling/expect.txt [hunk 4]
```diff
@@ -25,3 +26,4 @@ hpfvc
 knowledgebase
 pnp
 roadmap
+valildation
```
Comment (timja) [inline .github/actions/spelling/expect.txt:29]: this is misspelt

File: source/cloud-native-platform/path-to-live/public-dns.html.md.erb [hunk 5]
```diff
@@ -39,9 +39,22 @@ Here is an example [Pull Request](https://github.com/hmcts/azure-public-dns/pull
   record: "{copy endpoint hostname from frontdoor}"
 ```
 
-Where:
 
-`${env}`  =  Name of the environment for which you are creating the recordsets. Valid values are `sbox` or `prod`.
+## Verify custom domain on the Frontdoor
+- Create TXT records in the appropriate zone file for environments.
+Here is an example [Pull Request](https://github.com/hmcts/azure-public-dns/pull/1409/files),
```
Comment (timja) [inline source/cloud-native-platform/path-to-live/public-dns.html.md.erb:45]: you need to add the frontdoor config first to get the DNS records? 

I think the order should be:

1. FrontDoor config
2. DNS TXT records
3. CNAME / Alias records

should be 3 PRs (bit of a pain unfortunately)

**Review by timja — CHANGES_REQUESTED** [review]

File: source/cloud-native-platform/path-to-live/public-dns.html.md.erb (hunk 5 above)
Comment (cpareek) [reply source/cloud-native-platform/path-to-live/public-dns.html.md.erb:45]: think it should be, I have updated my PR now
1. CNAME / alias records
2. Frontdoor Config

File: source/cloud-native-platform/path-to-live/public-dns.html.md.erb [hunk 6]
```diff
@@ -41,8 +41,6 @@ Here is an example [Pull Request](https://github.com/hmcts/azure-public-dns/pull
 
 Where:
 
-`${env}`  =  Name of the environment for which you are creating the recordsets. Valid values are `sbox` or `prod`.
```
Comment (timja) [inline source/cloud-native-platform/path-to-live/public-dns.html.md.erb:44]: afdverify is no longer needed as well

File: source/cloud-native-platform/path-to-live/_front-door-config.md [hunk 7]
```diff
@@ -44,6 +45,7 @@
     name             = "app-name"
     custom_domain    = "your-app.service.gov.uk"
     backend_domain   = ["firewall-prod-int-palo-sdsprod.uksouth.cloudapp.azure.com"]
+    add_txt_record   = true
```
Comment (timja) [inline source/cloud-native-platform/path-to-live/_front-door-config.md:48]: shouldn't we just do this by default? or is there a problem with that?

**Review by timja — APPROVED** [review]
my questions can probably be iterated on although afdverify should really be fixed

File: source/cloud-native-platform/path-to-live/_front-door-config.md (hunk 7 above)
Comment (cpareek) [reply source/cloud-native-platform/path-to-live/_front-door-config.md:48]: just sent you PM on slack

**Pull request merged**